user-api/base/deploymentconfig.yaml

```diff
@@ -38,6 +38,11 @@ spec:
               value: "khemenu.thoth-station.ninja"
             - name: "THOTH_SENTRY_IGNORE_EXCEPTION"
               value: "thoth.user_api.exceptions.NotFoundException,thoth.storages.exceptions.NotFoundError"
+            - name: THOTH_USER_API_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  key: token
+                  name: thoth
```

Comment:
This does not exist. Not sure if it should be mapped to another key. CC @harshad16

Reply:
please use the key `thoth-secret` for this.

Reply:
Adjusted, PTAL